src/components/routes/Host/ListingForm/AccommodationsForm/AccommodationsForm.tsx

```diff
@@ -33,15 +34,16 @@ const AccommodationsForm = (props: any): JSX.Element => {
       </div> */}
```

Comment:
are we keeping the commented code for later?

Reply:
yea. However, I'm not sure how high pri adding additional fields is to our existing system and with the shifting business needs.